ts/features/euCovidCert/utils/screenshot.ts

```diff
@@ -1,18 +1,28 @@
+import path from "path";
 import { CaptureOptions, captureRef } from "react-native-view-shot";
 import { ReactInstance, RefObject } from "react";
 import { Dimensions } from "react-native";
+import RNFS from "react-native-fs";
 import { saveImageToGallery } from "../../../utils/share";
+import I18n from "../../../i18n";
 
-type CaptureScreenShotEvents = {
+type CaptureScreenshotEvents = {
   onSuccess?: () => void; // invoked on success
   onError?: () => void; // invoked on error
   onEnd?: () => void; // always invoked onSuccess/onError
   onNoPermissions?: () => void; // invoked in no given required permission
 };
 
-export const screenShotOption: CaptureOptions = {
+export type ScreenshotOptions = CaptureOptions & {
+  filename: string;
+  album?: string;
+};
+
+export const screenshotOptions: ScreenshotOptions = {
   width: Dimensions.get("window").width,
-  format: "png"
+  format: "png",
+  filename: I18n.t("features.euCovidCertificate.common.title"),
+  album: "IO"
```

Comment:
```suggestion
  album: I18n.t("features.euCovidCertificate.save.album") // it should be added in locales
```
Could it be a value set in locales instead of hard-coded string?
I believe it won't be necessary change it but if someone needs it for a similar flow we have a common value shared
What do you think?

Reply:
@Undermaken yeah, I thought about it. Generally speaking I was looking for a way to get the app's name (e.g. `app.json`), but I haven't found a good candidate. We can surely add it to locales.

Reply:
@Undermaken done! Please check that I added the localizable key correctly, thanks. I've also updated the unit tests accordingly.